package.yaml

```diff
@@ -90,11 +90,11 @@ library:
     - primitive
     - process
     - resource-pool
-    - retry
+    - retry >= 0.8.1.0
```

Comment:
We use `retryingDynamic`, introduced in this version. I discovered it because it's not present in `lts-12`.